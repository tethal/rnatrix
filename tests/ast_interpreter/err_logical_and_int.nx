fun main() {
    5 && true;
}
