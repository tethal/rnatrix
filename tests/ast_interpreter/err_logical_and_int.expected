<string>:1:1: error: expected a boolean value
5 && true;
^
